source/multiple-mongosyncs.txt

```diff
@@ -121,8 +111,12 @@ To configure multiple ``mongosync`` instances:
            --cluster1 "mongodb://user:password@cluster1host:27500" \
            --id shard01 --port 27601
 
+      When running multiple ``mongosync`` instances, you must specify the shard 
+      you want to replicate.  To do so, use the :option:`--id` option or the
```

Comment:
I wouldn't say "specify the shard you want to replicate". The makes it sound like they can replicate some but not all shards. The number of mongosync instances must be equal to the number of shards. And each mongosync should map to a source shard 1:1 via the "--id" option. Can we make this explicit?

Reply:
@ldennis.  Text adjusted.  Is it alright or would you like to see any other changes?